### ISSUE TITLE
Feature request: option to anchor the start of the track at the top of the donut

### ISSUE THREAD

**Issue by tobie** [simeydotme/donutty#38]
Is anyone working on adding an option to allow for anchoring the track at the top of the donut rather than the bottom?

**Comment by simeydotme** [comment]
@tobie It's not set as a feature request yet. If you desire that result, though, you could apply a transform (scaleY: -1) to achieve it :)

**Comment by tobie** [comment]
Wouldn't that turn any related text upside down, too?

**Comment by simeydotme** [comment]
yes @tobie , if you're using the text display feature too, you'll also need to invert that :)

**Comment by simeydotme** [comment]
here's an example;
https://codepen.io/simeydotme/pen/OJZqOxZ/880f03504427b58fc542cd4078429a44

**Comment by simeydotme** [comment]
If you want to submit a PR to make this an option, please go ahead :)

**Comment by simeydotme** [comment]
re-opened as @tobie is preparing a PR for the feature! 👍
